vendor/vendor.json

```diff
@@ -750,6 +750,14 @@
 			"version": "v1.14.33",
 			"versionExact": "v1.14.33"
 		},
+		{
+			"checksumSHA1": "nUdxsOW9jg+m+sWZYPu7oX9rZo8=",
+			"path": "github.com/aws/aws-sdk-go/service/kinesisanalytics",
```

Comment:
Can you please separate the vendoring into its own PR? We do this because these dependencies are a fast moving target, it makes the PRs a little more manageable, and we can quickly merge it in. FYI `master` is already AWS SDK Go `v1.15.4`.

https://github.com/terraform-providers/terraform-provider-aws/blob/2c3eaaef70430fa462a5750825b2a339e910348a/vendor/vendor.json#L41-L48

Reply:
Absolutely. I've created the PR here: https://github.com/terraform-providers/terraform-provider-aws/pull/5461